string name
string folder
---
bool success
trajectory_msgs/JointTrajectory trajectory_sent
